setup.py

```diff
@@ -0,0 +1,39 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="karoo_gp",
+    version="2.3",
```

Comment:
please change the version from `2.3` to `2.4` in all files and Ezio suggests using the __version__ function so as to change it across all files